Rct poster session news (#263)
iris-hep/iris-hep.github.io: _posts/2020-02-27-iris-hep-posters-2020.md

---
layout: irispost
title: IRIS-HEP Poster Session 2020 at Princeton University
author: Robert Tuck
postimage: /assets/images/posts/2020-02-27-iris-hep-posters-2020-main.jpg
postimage-caption: "Irina Espejo with the poster for Scalable Cyberinfrastructure Applications Photo Credit: Kyle Cranmer"
postbanner: /assets/images/posts/20200227-iris-hep-posters-2020-banner.jpg
summary: "IRIS-HEP held a Poster Session at Princeton University on February 27, 2020 to highlight the work of members from the first 18 months of IRIS-HEP"
---

{% include figure.html
    file="/assets/images/posts/20200227-iris-hep-posters-2020-Gray-ACTS.jpg"
    alt="Tracking with A Common Tracking Software (ACTS)"
    caption="Heather Gray with the poster for Tracking with A Common Tracking Software (ACTS) Photo Credit: Heather Gray."
%}

{% include figure.html
    file="/assets/images/posts/20200227-iris-hep-posters-2020-Schreiner-boost-histogram-extend-lhcb.jpg"
    alt="Extending the Physics Reach of LHCb in Run 3 and Boost-Histogram for Analysis System"
    caption="Henry Schreiner with posters for Extending the Physics Reach of LHCb in Run 3 Using Machine Learning in the

> Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis System Photo Credit: Henry Schreiner."

Reviewer: ```suggestion
Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis Systems. Photo Credit: Henry Schreiner."
```

%}

> On Thursday, February 27, 2020 members of IRIS-HEP got together at Princeton University to take stock of our progress in the first 18 months and plan for the future. A poster session was held in conjunction with this review session to highlight the work of members from the first 18 months of IRIS-HEP.

Reviewer: ```suggestion
On Thursday, February 27, 2020, members of IRIS-HEP got together at Princeton University to take stock of our progress in the first 18 months and plan for the future. A poster session was held in conjunction with this review session to highlight the work of members from the first 18 months of IRIS-HEP.
```

> In all, twenty seven posters were presented by PIs, Post Docs, Students, Fellows and Staff encompassing topics from all areas of IRIS-HEP.  The topics included general information about IRIS-HEP, operational and policy information,  the development and deployment of services and tools, and the application of novel approaches to computing challenges.

Reviewer: ```suggestion
In all, twenty-seven posters were presented by PIs, Postdocs, Students, Fellows, and Staff encompassing topics from all areas of IRIS-HEP.  The topics included general information about IRIS-HEP, operational and policy information,  the development and deployment of services and tools, and the application of novel approaches to computing challenges.
```
